app/models/question.rb

```diff
@@ -1,7 +1,11 @@
 class Question < ActiveRecord::Base
+  has_many :choices
   validates :body, presence: { message: "Question can't be blank" }
   validates :body, length: {
     maximum: 140,
     message: "Question must be less than 140 characters"
   }
+  accepts_nested_attributes_for :choices,
+    reject_if: proc { |attributes| attributes["description"].blank?
+  }
```

Comment:
`end` at 10, 2 is not aligned with `proc { |attributes| attributes["description"].blank?` at 9, 15 or `reject_if: proc { |attributes| attributes["description"].blank?` at 9, 4